SimPEG/EM/NSEM/Utils/dataUtils.py

```diff
@@ -54,7 +54,114 @@ def rotateData(NSEMdata, rotAngle):
 
     return Data.fromRecArray(outRec)
 
+# Function to get data and data info
+def extract_data_info(NSEMdata):
```

Comment:
This is a typical util, code to get data/survey info for assigning uncertainties. I used this but it is based on the older `np.recarray` data format that I am slowly facing out. I removed this function since it shouldn't be used.